packages/ibm-products/src/components/DataSpreadsheet/DataSpreadsheet.tsx

```diff
@@ -5,16 +5,17 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+import { ActiveCellCoordinates, Column, PrevState, Size, Theme } from './types';
```

Comment:
This line is causing some errors because there already a `Column` import coming from react-table. Can you import it as `ColumnType`?
```suggestion
import { ActiveCellCoordinates, Column as ColumnType, PrevState, Size, Theme } from './types';
```